pkgs/development/python-modules/cmdstanpy/default.nix

```
@@ -0,0 +1,77 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, cmdstan
+
+, pandas
+, numpy
+, tqdm
+, xarray
+
+, pytestCheckHook
+}:
+buildPythonPackage rec {
+  pname = "cmdstanpy";
+  version = "1.1.0";
```

```suggestion
  version = "1.1.0";
  format = "setuptools";
```
Please add a format explicitly.